modules/mapper-extras/src/test/java/org/elasticsearch/index/mapper/ScaledFloatFieldTypeTests.java

```diff
@@ -51,22 +50,6 @@ protected MappedFieldType createDefaultFieldType() {
         return ft;
     }
 
-    @Before
-    public void setupProperties() {
```

Comment:
I think we dropped this modifier accidentally.

Reply:
`scaling_factor` can't be updated, so we'd never be in a position when FieldTypeLookup would be comparing and old and new types with different scaling factors.

Reply:
👍